iscsidev/iscsi.go

```diff
@@ -115,6 +115,9 @@ func (dev *Device) CreateTarget() (err error) {
 	if err := iscsi.SetLunThinProvisioning(dev.targetID, TargetLunID); err != nil {
 		return err
 	}
+	if err := iscsi.DisableWriteCache(dev.targetID, TargetLunID); err != nil {
```

Comment:
NIT: Add a simple comment explaining why it disables  the write cache

Reply:
Added. Thank you.